Gopkg.toml

```diff
@@ -9,6 +13,10 @@ required = ["github.com/benbjohnson/tmpl","github.com/golang/protobuf/protoc-gen
   branch = "master"
   name = "github.com/davecgh/go-spew"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/mailru/easyjson"
```

Comment:
Can you add github.com/mailru/easyjson to the list of LICENSE_OF_DEPENDENCIES? Its MIT so no problem we just need to add it.